src/main/java/org/mskcc/cbio/oncokb/web/rest/UserUUIDController.java

```diff
@@ -73,6 +82,26 @@ public UserUUIDController(TokenProvider tokenProvider, AuthenticationManagerBuil
             Token token = tokenProvider.createTokenForCurrentUserLogin(Optional.empty(), Optional.empty());
             uuid = token.getToken();
         }
+
+        Optional<UserDetailsDTO> userDetails = userDetailsService.findByUserIsCurrentUser();
+        if (userDetails.isPresent()) {
+            UserDetailsDTO ud = userDetails.get();
+            boolean userHasTrialKey = 
```

Comment:
Try to use a method chaining. https://stackoverflow.com/questions/49380052/how-to-avoid-checking-for-null-values-in-method-chaining/49380103